camunda-formio-tasklist-vue/src/services/index.ts

```diff
@@ -0,0 +1,14 @@
+
+export * from "../models/sortOptions";
```

Comment:
Shouldn't TaskListSortOption be imported from `models/index.ts` file for consistency?